runtime/opt/zalando/runtime/Docker

```diff
@@ -29,6 +30,15 @@ def get_env_options(config: dict):
         yield '-e'
         yield 'ETCD_URL=http://172.17.42.1:2379'
 
+    yield '-e'
+    yield 'APPLICATION_ID={}'.format(config.get('application_id'))
+
+    yield '-e'
+    yield 'APPLICATION_VERSION={}'.format(config.get('application_version'))
+
+    yield '-e'
+    yield 'INSTANCE_ID={}'.format(boto.utils.get_instance_metadata()['instance-id'])
```

Comment:
Since these information are all an API for apps, it would also bind us to AWS which we currently do not.